packages/ethers-provider/package.json

```diff
@@ -29,6 +29,14 @@
     "rxjs": "7.5.6"
   },
   "peerDependencies": {
```

Comment:
Why don't you use specific versions of the packages?

Reply:
It's peer dependencies that are actually required on the app itself, not this lib.
I had to add them to the package.lock for `lib/fallback-provider.ts` file to be bundled using rollup cli, but these dependencies must be ignored, otherwise it will not work.

Both peer dependencies and fallback provider file is supposed to be removed later, once ethers will merge their fix to their repo